docs/source/models.rst

```diff
@@ -81,6 +81,98 @@ time_created
 run_id
     ID of the run that created the model, if the model was saved using :ref:`tracking`.
 
+signature
+  model signature (see below) in JSON format.
+
+input_example
+  reference to an artifact with input example (see below).
+
+
+.. _model-metadata:
+
+Model Metadata
+--------------
+When working with ML models you often need to know some basic functional properties of the model
+at hand, such as "What inputs does it expect?" and "What output does it produce?". MLflow models can
+include the following additional metadata that can be used by downstream tooling:
```

Comment:
Maybe a small table of contents section here would be helpful here to quickly indicate that the pieces of metadata are `Model Signature` and `Model Input Example` , since the`Model Signature` section is somewhat long.